test/functional/grade_entry_forms_controller_test.rb

```diff
@@ -412,7 +416,8 @@ class GradeEntryFormsControllerTest < AuthenticatedControllerTest
                     :description => @grade_entry_form.description,
                     :message => @grade_entry_form.message,
                     :date => @grade_entry_form.date,
-                    :grade_entry_items_attributes => {'1' => @q1, '2' => @q2}}}
+                    :grade_entry_items_attributes => {'1' => @q1, '2' => @q2},
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.